boards/arm/frdm_kl46z/doc/index.rst

```diff
@@ -0,0 +1,235 @@
+.. _frdm_kl46z:
+
+NXP FRDM-KL46Z
+##############
+
+Overview
+********
+
+The Freedom KL46Z is an ultra-low-cost development platform enabled by the
+Kinetis |reg| L series KL4x MCU family built on the Arm |reg| Cortex |reg|-M0+
+processor.
+
+
+.. image:: ./frdm_kl46z.jpg
```

Comment:
so close, rename the image file to have a ``jpg`` extension to match this reference (not a ``jpeg`` as it has now).